### PR TITLE
fix(forms-web-app): adds packages/forms-web-app/src/public/javascript…

### PR DESCRIPTION
**Pull request by a6software**
…s to global gitignore

## Ticket Number

AS-1230

## Description of change
adds packages/forms-web-app/src/public/javascripts to global gitignore

## Checklist

- [ ] Requires infrastructure changes
- [ ] If adding or remove environment variables (e.g. in `docker-compose.yaml`) then I have updated the appropriate Helm chart
- [ ] I have updated the documentation accordingly
- [ ] My commit history in this PR is linear
- [ ] New features have tests
- [ ] Breaking change (team conversation required)

## Important

Please do not merge from `master` (please only [rebase](https://github.com/foundry4/appeal-planning-decision/wiki/An-intro-to-Git-Rebase)). This keeps the history linear and easier to debug.

### CODE REVIEW

**Review by davehlewis — DISMISSED** [review]

File: .gitignore [hunk 1]
```diff
@@ -171,6 +171,7 @@ tmp
 known_hosts
 
 packages/**/globalConfig.json
+packages/forms-web-app/src/public/javascripts
```
Comment (MrSimonEmms) [inline .gitignore:174]: Is it worth doing this as a wildcard in case we need to do the same for LPA Questionnaire?

**Review by MrSimonEmms — APPROVED** [review]
Happy with this, with just one question

File: .gitignore (hunk 1 above)
Comment (a6software) [reply .gitignore]: Done

**Review by rcmsoftwaredevelopment — APPROVED** [review]

**Comment by a6software** [comment]
ty

**Pull request merged**